### DISCUSSION TITLE
How can I select the SSH port?

### DISCUSSION THREAD

**Discussion by vivamoto** [winfsp/winfsp#361] [Q&A]
Hi,

I created a tunnel from my Windows computer to my Linux server with direct connection (without password). The tunnel uses port 2022. How can I mount the driver using this port? Windows asks for password when I mount the remote drive.

Best regards,
Victor.

**Comment by billziss-gh** [comment]
This should probably be asked over at [SSHFS-Win](https://github.com/billziss-gh/sshfs-win).

SSHFS-Win allows you to use a port when mounting. See this syntax: `\\sshfs\REMUSER@HOST[!PORT][\PATH]`
